slices/coreutils.yaml

```diff
@@ -1,133 +1,352 @@
+# This package contains the basic file, shell and text manipulation utilities
+# which are expected to exist on every operating system.
 package: coreutils
 
 essential:
   - coreutils_copyright
 
+# The coreutils package contains three standard data slices:
+#   - libs: contains all shared object files.
+#   - bins: contains all binaries, or executables.
+#   - copyright: contains the copyright file.
+# It also contains a list of other granular slices, each of which contains a
+# list of binaries. These slices have been created based on the grouping in the
+# GNU docs: https://www.gnu.org/software/coreutils/manual/html_node/index.html.
+# Ideally, users would install (or, add as an essential to some other slice) one
+# or more of these granular slices when they need a particular set of binaries.
 slices:
   libs:
     essential:
       - libc6_libs
     contents:
       /usr/libexec/coreutils/libstdbuf.so:
 
+  # The bins slice contains all of the binaries in the coreutils package.
   bins:
     essential:
-      - coreutils_libs
-      - libacl1_libs
-      - libattr1_libs
+      - coreutils_basic-ops
+      - coreutils_char-ops
+      - coreutils_chattr
+      - coreutils_cmd-invocation
+      - coreutils_conditions
+      - coreutils_delaying
+      - coreutils_dir-listing
+      - coreutils_field-ops
+      - coreutils_file-parts
+      - coreutils_fname-manip
+      - coreutils_formatting
+      - coreutils_numeric
+      - coreutils_printing
+      - coreutils_redirection
+      - coreutils_rw-entire-files
+      - coreutils_selinux
+      - coreutils_sort-ops
+      - coreutils_space-usage
+      - coreutils_special-types
+      - coreutils_summarizing
+      - coreutils_sys-info
+      - coreutils_user-info
+      - coreutils_working-context
```

Comment:
```suggestion
      - coreutils_basic-operations
      - coreutils_operating-on-characters
      - coreutils_changing-file-attributes
      - coreutils_modified-command-invocation
      - coreutils_conditions
      - coreutils_delaying
      - coreutils_directory-listing
      - coreutils_operating-on-fields
      - coreutils_output-of-parts-of-files
      - coreutils_file-name-manipulation
      - coreutils_formatting-file-contents
      - coreutils_numeric-operations
      - coreutils_printing-text
      - coreutils_redirection
      - coreutils_output-of-entire-files
      - coreutils_selinux-context
      - coreutils_operating-on-sorted-files
      - coreutils_file-space-usage
      - coreutils_special-file-types
      - coreutils_summarizing-files
      - coreutils_system-context
      - coreutils_user-information
      - coreutils_working-context
```

I think we should stick to a well-defined rationale for the names. Atm we have `ops` as a shortening for `operations`, and then hide `ops` for some slices, like `numeric`; as well as using other hard-to-predict short names like `cmd` or `sys`, and `rw-entire-files` vs `file-parts`.

What about the proposal above (out of order though)?

Reply:
Pushed https://github.com/canonical/chisel-releases/pull/339/commits/e9a447c9e8348e1cfeff5226c343ee98d0cd1999. Thanks.